hw/top_earlgrey/data/pins_cw310.xdc

```diff
@@ -59,13 +59,13 @@ set_property -dict { PACKAGE_PIN A15  IOSTANDARD LVCMOS33 } [get_ports { IOB2  }
 set_property -dict { PACKAGE_PIN F14  IOSTANDARD LVCMOS33 } [get_ports { IOB3  }]; #  F14:USR_SPI1CS
 set_property -dict { PACKAGE_PIN AE25 IOSTANDARD LVCMOS33 } [get_ports { IOB4  }]; # AE25:CWIO_IO1     EarlGrey:UART1_RX   ChipWhisperer_ConnectorJ14_PIN10
 set_property -dict { PACKAGE_PIN AF25 IOSTANDARD LVCMOS33 } [get_ports { IOB5  }]; # AF25:CWIO_IO2     EarlGrey:UART1_TX   ChipWhisperer_ConnectorJ14_PIN12
-set_property -dict { PACKAGE_PIN N23  IOSTANDARD LVCMOS33 } [get_ports { IOB6  }]; #  N23:PMOD2_IO5    EarlGrey:GPIO       (BoB):PMOD1_CSB7
-set_property -dict { PACKAGE_PIN N26  IOSTANDARD LVCMOS33 } [get_ports { IOB7  }]; #  N26:PMOD2_IO6    EarlGrey:GPIO       (BoB):PMOD1_RST
-set_property -dict { PACKAGE_PIN M20  IOSTANDARD LVCMOS33 } [get_ports { IOB8  }]; #  M20:PMOD2_IO7    EarlGrey:I2C0_SCL   (BoB):PMOD1_I2C_SCL
-set_property -dict { PACKAGE_PIN P25  IOSTANDARD LVCMOS33 } [get_ports { IOB9  }]; #  P25:PMOD2_IO8    EarlGrey:I2C0_SDA   (BoB):PMOD1_I2C_SDA
-set_property -dict { PACKAGE_PIN V9   IOSTANDARD LVCMOS18 } [get_ports { IOB10 }]; #   V9:USRDIP4                          DIP switches
-set_property -dict { PACKAGE_PIN W8   IOSTANDARD LVCMOS18 } [get_ports { IOB11 }]; #   W8:USRDIP5                          DIP switches
-set_property -dict { PACKAGE_PIN W10  IOSTANDARD LVCMOS18 } [get_ports { IOB12 }]; #  W10:USRDIP6                          DIP switches
+set_property -dict { PACKAGE_PIN U9   IOSTANDARD LVCMOS33 } [get_ports { IOB6  }]; #   U9:USRDIP0                          DIP switches
+set_property -dict { PACKAGE_PIN N23  IOSTANDARD LVCMOS33 } [get_ports { IOB7  }]; #  N23:PMOD2_IO5    EarlGrey:GPIO       (BoB):PMOD1_CSB7
+set_property -dict { PACKAGE_PIN N26  IOSTANDARD LVCMOS33 } [get_ports { IOB8  }]; #  N26:PMOD2_IO6    EarlGrey:GPIO       (BoB):PMOD1_RST
+set_property -dict { PACKAGE_PIN V7   IOSTANDARD LVCMOS33 } [get_ports { IOB9  }]; #   V7:USRDIP1                          DIP switches
+set_property -dict { PACKAGE_PIN V8   IOSTANDARD LVCMOS18 } [get_ports { IOB10 }]; #   V8:USRDIP2                          DIP switches
+set_property -dict { PACKAGE_PIN M20  IOSTANDARD LVCMOS18 } [get_ports { IOB11 }]; #  M20:PMOD2_IO7    EarlGrey:I2C0_SCL   (BoB):PMOD1_I2C_SCL
+set_property -dict { PACKAGE_PIN P25  IOSTANDARD LVCMOS18 } [get_ports { IOB12 }]; #  P25:PMOD2_IO8    EarlGrey:I2C0_SDA   (BoB):PMOD1_I2C_SDA
```

Comment:
Unfortunately this arrangement puts SCL/SDA on a 1V8 bank. Is there another configuration that might work without too much churn?

Reply:
Oh, that was just an accident--I didn't update the IOSTANDARD. This bank is 3.3 V.

Reply:
Fixed.

Reply:
oh great, ezpz :)